lib/ui/page/home/page/chat/message_field/view.dart

```diff
@@ -896,7 +896,11 @@ class MessageFieldView extends StatelessWidget {
         mainAxisSize: MainAxisSize.min,
         children: [
           const SizedBox(width: 12),
-          SvgLoader.asset('assets/icons/edit.svg', width: 17, height: 17),
+          SvgImage.asset(
+            'assets/icons/edit.svg',
+            width: 17,
+            height: 17,
```

Comment:
И тут будет ли форматирование лучше?

```suggestion
            height: 17
```

Reply:
У нас большинство таких ассетов, в которых имеется >2 параметров, выполнены через запятую, потому что они длинные, чисто теоретически оно встает там со всеми параметрами без переносов, но я думаю в перспективе общей картины лучше будет оставить запятую.